cmd/dagger/call.go

```diff
@@ -114,9 +114,6 @@ dagger call lint stdout
 		if silent || !(progress == "auto" && autoTTY || progress == "tty") {
 			return fmt.Errorf("running shell without the TUI is not supported")
 		}
-		if debug {
-			return fmt.Errorf("running shell with --debug is not supported")
-		}
```

Comment:
This works fine now; the debug output will be printed on exit. In the future I want to make it some kind of tabbed view so you can start a REPL and tab over to view progress.